src/transport/ipc.rs

```diff
@@ -15,8 +15,8 @@ use crate::ZmqResult;
 use futures::{select, FutureExt};
 use std::path::{Path, PathBuf};
 
-pub(crate) async fn connect(path: PathBuf) -> ZmqResult<(FramedIo, Endpoint)> {
-    let raw_socket = UnixStream::connect(&path).await?;
+pub(crate) async fn connect(path: &PathBuf) -> ZmqResult<(FramedIo, Endpoint)> {
```

Comment:
If we don't need ownership of the `PathBuf`, then this should just be an `&Path`

Reply:
Thanks for the hint. I'll fix it

Reply:
Done